src/main/java/com/mateacademy/listener/MyListener.java

```diff
@@ -0,0 +1,38 @@
+package com.mateacademy.listener;
+
+import org.apache.log4j.Logger;
+
+import javax.persistence.*;
```

Comment:
Please don't use a wild card in import. 
https://stackoverflow.com/questions/147454/why-is-using-a-wild-card-with-a-java-import-statement-bad